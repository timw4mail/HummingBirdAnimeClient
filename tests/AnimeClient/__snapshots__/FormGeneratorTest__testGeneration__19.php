<?php return '<select id="type" name="type">
    <option value="mysql">MySQL</option>
    <option value="pgsql">PostgreSQL</option>
    <option value="sqlite">SQLite</option>
</select>
';
