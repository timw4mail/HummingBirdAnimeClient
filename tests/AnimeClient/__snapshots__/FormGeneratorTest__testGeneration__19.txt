<select id="type" name="type">
    <option value="mysql">MySQL</option>
    <option value="pgsql">PostgreSQL</option>
    <option value="sqlite" selected>SQLite</option>
</select>
